portal/src/components/CodeBlock/CodeBlock.scss

```diff
@@ -0,0 +1,37 @@
+@import "~@fremtind/jkl-core/_functions.scss";
+@import "~@fremtind/jkl-core/variables/_all.scss";
+
+.jkl-portal-code-block {
+    position: relative;
+    background-color: $gra-20;
+    border-radius: rem(2px);
+    width: 100%;
+    max-width: rem(768px);
```

Comment:
Synes jag ser dessa widths/breakpoints lite överallt, vore det ikke bra att ha dem som variabler?

Reply:
Jo, vi burde finne ut hva som er "riktig" verdi her og bruke den alle steder, også for bredde på avsnitt, etc. Tror det bør være et eget issue i så fall.